packages/block-library/src/group/index.js

```diff
@@ -29,8 +29,10 @@ export const settings = {
 	],
 	example: {
 		attributes: {
-			customBackgroundColor: '#ffffff',
-			customTextColor: '#000000',
+			style: {
```

Comment:
Do you know why this example differs from the actual usage?:
![Screen Shot 2020-03-24 at 11 00 00](https://user-images.githubusercontent.com/699132/77412772-a18fef80-6dbe-11ea-80bb-aa7c4145947e.png)

Reply:
Outdated example :)